src/jit/codegen.h

```diff
@@ -465,11 +465,11 @@ protected :
                                     IL_OFFSETX                                    ilOffset);
 
 
+#if defined(_TARGET_ARM_)
     //
```

Comment:
Changes like this are unfortunate, because it implies that the comment applies only to the things within the #if/#endif, which is not true.